### ISSUE TITLE
[command:version] Version number in related peerDependencies don't update

### ISSUE THREAD

**Issue by BioPhoton** [lerna/lerna#2402]
When run lerna version also respect dependencies in `peerDependencies`.

## Current Behavior
When execution lerna version the peerDependencies don't update, but the devDependencies do.
Here a result after the bumping version of packageA and packageB from `0.0.0` to `0.0.1`

**packageA**
```json
{
  "name": "packageA",
  "version": "0.0.1",
}
```
**packageB**
```json
{
  "name": "packageB",
  "version": "0.0.1",
  "peerDependencies": {
    "@scope/packageA": "0.0.0"
  },
  "devDependencies": {
    "@scope/packageA": "0.0.1"
  }
}
```

## Expected Behavior

**packageA**
```json
{
  "name": "packageA",
  "version": "0.0.1",
}
```
**packageB**
```json
{
  "name": "packageB",
  "version": "0.0.1",
  "peerDependencies": {
    "@scope/packageA": "0.0.1"
  },
  "devDependencies": {
    "@scope/packageA": "0.0.1"
  }
}
```

**Comment by ValeryVS** [comment]
It isn't always the right way.

May be `packageB` can work with `packageA@^15.5.1` while recent `packageA` is `15.7.0`.
Or, it can even depend on `^1 | ^2 | ^3`. Such peerDependencies is quite common.

**Comment by arsnl** [comment]
@ValeryVS true. Peer dependencies don't act the same as dev nor prod dependencies. However, in some case the peerDependencies should follow the current version. Maybe we should add a new option to the version command to cover this need? Something like `--update-peer-dependencies`.

**Comment by ghiscoding** [comment]
hey guys, I'm looking at implementing this as an option in [Lerna-Lite](https://github.com/ghiscoding/lerna-lite), it will be opt-in (so disabled by default) and it will be very restrictive (it will only work with regular versions). I got an open PR [here](https://github.com/ghiscoding/lerna-lite/pull/362), feel free to provide feedback on it. I expect to release it within a week or so

It will update (bump) regular semver like these
- `1.2.3`
- `^1.2.3`
- `~1.2.3`
- `workspace:^1.2.3`

but it won't mutate/update things like
- `>=1.0.0`
- `>=1.0.0 < 2.0.0`
- `^1 | ^2 | ^3`

**Comment by RS-Sautter** [comment]
@JamesHenry can we add this feature also to lerna? If necessary I could help with that.

**Comment by runspired** [comment]
An OSS project I work on with many packages in a mono-repo with peer-deps on each other moved to strictly declaring peerDependencies recently now that most package managers properly support and respect them. Unfortunately, this results in peerDependencies being incorrect. We can manually fix the versions during our release process; however, it makes lerna's `version` command not all that useful since the commit and tag would not include this change.

**Comment by zimtsui** [comment]
As to Lerna 6.5.1, `lerna version` will stuck if there are any peerdependencies among local packages within the monorepo.

There is an simple example: <https://github.com/zimtsui/lerna-bug>

**Comment by andrii-bodnar** [comment]
Experiencing the same issue in the [Lingui](https://github.com/lingui/js-lingui) project. This project contains a couple of packages and some of them have peer dependencies to others. As a result, the `lerna version` command doesn't update these dependencies

**Comment by RS-Sautter** [comment]
ping, any news? Would a MR be accepted?
